lib/core.js

```diff
@@ -702,7 +702,7 @@ declare var exports: any;
 
 /* Commonly available, shared between node and dom */
 declare var console: {
-  assert(condition: boolean, ...data: Array<any>): void;
+  assert(condition: any, ...data: Array<any>): void;
```

Comment:
https://flowtype.org/docs/builtins.html#mixed

It's just more specific about what the input is here (even if that doesn't catch any real world errors). In general I would just think of `any` as a quick lazy solution, and always prefer `mixed`.